gcc/d/patches/patch-targetdm-8.x

```diff
@@ -511,6 +511,17 @@ These official OS versions are not implemented:
 +  d_add_builtin_version ("AArch64");
 +  d_add_builtin_version ("D_HardFloat");
 +}
+--- a/gcc/config/aarch64/aarch64-linux.h
++++ b/gcc/config/aarch64/aarch64-linux.h
+@@ -80,6 +80,8 @@
+     }						\
+   while (0)
+ 
++#define GNU_USER_TARGET_D_CRITSEC_SIZE 48
++
```

Comment:
Not sure how `TARGET_D_CRITSEC_SIZE` is evaluated, but it seems strange that it's defined as a simple number here but the name of a  function for other architectures.

Wouldn't you have to use `TARGET_D_CRITSEC_SIZE()` to make functions work? But does `constant_integer()` work in C++?